crates/storage/src/transactional.rs

```diff
@@ -656,6 +664,68 @@ mod test {
         use super::*;
         use crate::column::Column;
 
+        #[test]
+        fn read_returns_from_view_exact_size() {
+            // setup
+            let storage = InMemoryStorage::<Column>::default();
+            let mut view = storage.read_transaction();
+            let key = vec![0xA, 0xB, 0xC];
+            let expected = Value::from([1, 2, 3]);
```

Comment:
nit: we probably shouldn't use `expected` here (**or on the other tests**). Copy pasta.
Perhaps `source_data` makes more sense.

Reply:
should we address in a follow up?

Reply:
done